### DISCUSSION TITLE
missing manual file on linux

### DISCUSSION THREAD

**Discussion by freddii** [espanso/espanso#1205] [Feature requests and Ideas]
normally you are able to read a manual file when you type in terminal:
`man programname`

But it seams there is no manual for espanso.

It would be very handy to have a manual file right from terminal.

**Comment by Scrumplex** [comment]
*just sliding in so I can keep track of this for the AUR package*

**Comment by federico-terzi** [comment]
Hey guys,

Yes, this could be a good idea. Thank you for pointing this out!

Cheers :)

**Comment by AucaCoyan** [comment]
Regarding https://github.com/espanso/espanso/discussions/1934 we are moving back discussions into issues (only have issues 🙏🏼 ).
I like this feature request, so I'll create an issue 😄
